addon/middleware.js

```diff
@@ -0,0 +1,154 @@
+/* global Ember */
+import handlers from './handlers';
+import tracker  from './tracker';
+
+export default {
+  use: function(addon) {
+    var tracker = tracker.build(addon);
```

Comment:
I'm getting an error in console: `Uncaught TypeError: Cannot read property 'build' of undefined`
I believe this is because local `tracker` variable overwrites imported `tracker` variable